test/Parse/ConditionalCompilation/basicParseErrors.swift

```diff
@@ -148,7 +148,7 @@ undefinedFunc() // expected-error {{use of unresolved identifier 'undefinedFunc'
 #if _endian(arm64) // expected-warning {{unknown endianness for build configuration '_endian'}} expected-note {{did you mean 'arch'}} {{5-12=arch}}
 #endif
 
-#if targetEnvironment(_ObjC) // expected-warning {{unknown target environment for build configuration 'targetEnvironment'}} expected-note {{did you mean 'simulator'}} {{23-28=simulator}}
+#if targetEnvironment(_ObjC) // expected-warning {{unknown target environment for build configuration 'targetEnvironment'}} expected-note {{did you mean 'macabi'}} {{23-28=macabi}}
```

Comment:
Hmm, it probably shouldn't suggest "macabi" since that's deprecated in favor of "macCatalyst"... (This can just be a bug report, though.)

Reply:
I'd like the fix this separately. But I agree it needs to be fixed.